# H_ovalis_supplementary

#This script "calculate_blast_coverage.py" calculates the horizontal coverage of CDS based on dcmegablast results of reads aligned to reference genome

#USAGE:
#python calculate_blast_coverage.py <reference genome CDS in fasta> <dcmegablast hit table>

#This script was included in the following publications:
#1. Golicz AA, Schliep M, Lee HT, Larkum AW, Dolferus R, Batley J, Chan CK, Sablok G, Ralph PJ, Edwards D. 2015. Genome-wide survey of the seagrass Zostera muelleri suggests modification of the ethylene signalling network. Journal of Experimental Botany 66, 1489-1498.
#2. Lee HT, Golicz AA, Bayer PE, Severn-Ellis A, Chan CK, Batley J, Kendrick GA, Edwards D. 2018. Genomic comparision of two independent seagrass lineages reveals habitat-driven convergent evolution. Journal of Experimental Botany <manuscript submitted>
